hapi/hedera-protobufs/services/consensus_create_topic.proto

```diff
@@ -70,4 +71,37 @@ message ConsensusCreateTopicTransactionBody {
      * If specified, there must be an adminKey and the autoRenewAccount must sign this transaction.
      */
     AccountID autoRenewAccount = 7;
+
+    /**
+     * Access control for update/delete of custom fees.
+     * <p>
+     * If unset, custom fees CANNOT be set for this topic.<br/>
+     * If not set when the topic is created, this field CANNOT be set via update.
+     * If set when the topic is created, this field MAY be changed via update.
+     */
+    Key fee_schedule_key = 8;
+
+    /**
+     * A set of keys that are allowed to submit messages to the topic without
+     * paying the topic's custom fees.
+     * <p>
+     * If a submit transaction is signed by _any_ key included in this set,
+     *  custom fees SHALL NOT be charged for that transaction.
+     * <p>
+     * If fee_exempt_key_list is unset, the following keys are exempt
+     *  from custom fees: adminKey, submitKey, fee_schedule_key.
```

Comment:
For clarity, we should make this express the HIP _intent_:
```suggestion
     * If fee_exempt_key_list is unset in this transaction, the following
     * keys SHALL be added to this list:
     * `adminKey`, `submitKey`, `fee_schedule_key`.
```

Reply:
Fixed. Thanks for the suggestion.